### PR TITLE
Kafka Schemas

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: kafka_schemas.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+	}
+
+	var r KafkaSchemaValidateResponse
+	errR := checkAPIResponse(bts, &r)
+
+	return r.IsCompatible, errR
+}
+
+// AddSubject adds a new kafka Schema
+func (h *KafkaSchemaHandler) AddSubject(project, service, name string, subject KafkaSchemaSubject) (*KafkaSchemaSubjectResponse, error) {
```
Comment (mteiste) [inline kafka_schemas.go]: need to think about these names, AddSubject is a bad name for something that actually Adds a schema

File: kafka_schemas.go [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+				return nil, err
+			}
+
+			if !isValid {
+				return nil, errors.New("kafka schema is not compatible with version :" + strconv.Itoa(hVersion))
+			}
+		}
+	}
+
+	println("hi again")
```
[the remark this answers is not in the document]
Comment (ivan-savciuc) [reply kafka_schemas.go]: ups, yes

File: kafka_schemas.go (hunk 1 above)
Comment (ivan-savciuc) [reply kafka_schemas.go]: sounds good

File: kafka_schemas.go [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+	path := buildPath("project", project, "service", service, "kafka", "schema", "subjects", name, "versions")
+	println(path)
```
[the remark this answers is not in the document]
Comment (ivan-savciuc) [reply kafka_schemas.go]: yes, thanks

File: kafka_schemas.go [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+		APIResponse
+		IsCompatible bool `json:"is_compatible"`
+	}
+)
+
+func NewKafkaSchema(s string) KafkaSchemaSubject {
+	space := regexp.MustCompile(`\s+`)
+
+	return KafkaSchemaSubject{
+		Schema: string(space.ReplaceAllString(s, "")),
```
[the remark this answers is not in the document]
Comment (ivan-savciuc) [reply kafka_schemas.go]: wanted to make JSON payload smaller, but probably without a good reason, I think we do not need this NewKafkaSchema at all

File: CHANGELOG.md [hunk 5]
```diff
@@ -18,3 +18,7 @@ Add support for cross region vpc peering.
 ## [1.3.0] - 2019-12-23
 
 Add support for Elasticsearch ACLs.
+
+## [1.3.0] - 2020-01-03
```
Comment (rikonen) [inline CHANGELOG.md]: Same version repeated

Comment (ivan-savciuc) [reply CHANGELOG.md]: fixed

**Pull request merged**